docs/en_US/regions.md

```diff
@@ -0,0 +1,14 @@
+# Unsupported Regions
+
+Pretendo does not support access from certain countries due to technical and legal limitations. These restrictions are primarily due to Cloudflare's network limitations and other compliance factors.
+
+## Countries Not Supported:
+- **China** 🇨🇳 (Great Firewall restrictions, Cloudflare limitations)  
+- **Russia** 🇷🇺 (Cloudflare limitations, regulatory concerns)  
+- **Iran** 🇮🇷 (Cloudflare limitations, sanctions)  
+- **North Korea** 🇰🇵 (Sanctions, lack of internet access)  
+- **Syria** 🇸🇾 (Cloudflare limitations, sanctions)  
+- **Belarus** 🇧🇾 (Cloudflare limitations, regulatory concerns)  
+- **Turkmenistan** 🇹🇲 (Strict internet censorship, Cloudflare limitations)  
+
+If you experience any errors outside of these countries check out the unofficial [Pretendo Status Website](https://status.pretendo.zip).
```

Comment:
While this site is ran by a moderator on our Discord and is used by the community, as noted this is not an official site we support. So it shouldn't be on our website